src/server/watchGuard/watchGuard.ts

```diff
@@ -0,0 +1,9 @@
+/// <reference types="node" />
+
+if (process.argv.length < 3) {
+    process.exit(1);
+}
+const directoryName = process.argv[2];
+const fs: { watch(directoryName: string, options: any, callback: () => {}): any } = require("fs");
+fs.watch(directoryName, { recursive: true }, () => ({})).close();

```

Comment:
Does the libuv bug definitely cause the process to crash synchronously on calling "fs.watch"? (i.e. it's not going to get to `process.exit(0)` before it the crash would occur).